crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -167,6 +185,18 @@ pub(crate) enum TokenKind {
     /// `.`.
     Dot,
 
+    /// `else`
+    Else,
+
+    /// `if`
+    If,
+
+    /// `in`
+    In,
+
+    /// `match`
+    Match,
+
     /// Any other non trivia token. Always has a length of 1
```

Comment:
oops i guess this is no longer true (length 1)

Reply:
Good find